index.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <link rel="stylesheet" href="css/main.css">
+        <!-- <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous"> -->
+        <title>To Do App</title>
+    </head>
+    <body>
+        <Center>
+            <div id="vueApp">
+                <h1 class="title" v-text="title"></h1> 
+                    <div class="container">
+                        <input v-model="newItem" v-on:keyup.enter="addItem"/>
+                        <span>
+                            <button @click.prevent="addItem">Add</button> 
+                            <button @click.prevent="clearState">Clear All Tasks</button>
+                        </span>
+                        <ul v-for="list in lists" vbind:key="list.id" style="list-style-type:none;">
+                            <li><input type="checkbox" v-model="list.completed" @click="setComplete(list.id, this)">{{list.item}}<button @click.prevent="removeItem(list.id)">Delete</button></li>
```

Comment:
```suggestion
                            <li><input type="checkbox" v-model="list.completed" @click="setComplete(list.id)">{{list.item}}<button @click.prevent="removeItem(list.id)">Delete</button></li>
```
I'm not sure if you need the `this` parameter here.